docs/de/docs/index.md

```diff
@@ -107,16 +102,16 @@ The key features are:
 
 If you are building a <abbr title="Command Line Interface">CLI</abbr> app to be used in the terminal instead of a web API, check out <a href="https://typer.tiangolo.com/" class="external-link" target="_blank">**Typer**</a>.
 
-**Typer** is FastAPI's little sibling. And it's intended to be the **FastAPI of CLIs**. ⌨️ 🚀
+**Typer** ist das kleine Schwesterchen von FastAPI. Und es soll die **FastAPI der CLIs** sein. ⌨️ 🚀
```

Comment:
I would suggest "Typer ist FastAPIs kleine Schwester." as I believe it sounds better